src/ExpressionEvaluator/Core/Source/ResultProvider/IDkmClrFormatter2.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.ObjectModel;
+using Microsoft.VisualStudio.Debugger.Evaluation;
+using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
+
+namespace Microsoft.VisualStudio.Debugger.ComponentInterfaces
+{
+    public interface IDkmClrFormatter2
```

Comment:
Interfaces do not inherit in the debugger API. And inheritance should not be required here in the implementation.